setup.py

```diff
@@ -66,16 +64,12 @@
     # >= 0.4.0 to be compatible with spaCy 3.
     "scispacy>=0.4.0",
     "sentence-transformers",
+    "setuptools>=45",
```

Comment:
I'm curious what lead to this requirement?

Would it also be possible to move this to `pyproject.toml` because I imging this is only required at install time but not at run time.

Reply:
Yes, clearly this `setuptools` didn't belong here, I removed it in a following commit.

I was mainly trying things around to fix an RTD error related to inconsitent `setuptools` version.

I added more details in my comment [here](https://github.com/BlueBrain/Search/pull/388#issuecomment-869686029).